core/src/components/card/card.tsx

```diff
@@ -7,7 +7,7 @@ import { generateUniqueId, hasSlot } from '../../utils/utils';
  * @slot thumbnail - Slot for the Card thumbnail.
  * @slot body - Slot for the body section of the Card.
  * @slot body-image - Slot for the body section of the Card, used for image.
- * @slot bottom - Slot for the bottom section of the Card.
```

Comment:
Link part disappeared completely now, is there maybe some place that still calls for the "bottom" slot in Card component?
![image](https://github.com/scania-digital-design-system/tegel/assets/48509966/ffb3aa95-8d2f-4dea-a0a2-d6607d5a6712)